BLOCKFILE

```diff
@@ -33,6 +31,48 @@ hw/vendor/lowrisc_ibex/rtl/*
 hw/vendor/pulp_riscv_dbg/src/*
 hw/vendor/pulp_riscv_dbg/debug_rom/*
 
+# IP block RTL for blocks that require a higher scruitinty on changes to
+# maintain stablity (Ascon, DMA and Key Manager DPE are not blocked).
+hw/ip/adc_ctrl/rtl/*
+hw/ip/aes/rtl/*
+hw/ip/aon_timer/rtl/*
+hw/ip/csrng/rtl/*
+hw/ip/edn/rtl/*
+hw/ip/entropy_src/rtl/*
+hw/ip/gpio/rtl/*
+hw/ip/hmac/rtl/*
+hw/ip/i2c/rtl/*
+hw/ip/keymgr/rtl/*
+hw/ip/kmac/rtl/*
+hw/ip/lc_ctrl/rtl/*
+hw/ip/otbn/rtl/*
+hw/ip/otp_ctrl/rtl/*
+hw/ip/pattgen/rtl/*
+hw/ip/pinmux/rtl/*
+hw/ip/prim/rtl/*
+hw/ip/prim_generic/rtl/*
+hw/ip/prim_xilinx/rtl/*
+hw/ip/prim_xilinx_ultrascale/rtl/*
+hw/ip/pwm/rtl/*
+hw/ip/rom_ctrl/rtl/*
+hw/ip/rv_core_ibex/rtl/*
+hw/ip/rv_dm/rtl/*
+hw/ip/rv_timer/rtl/*
+hw/ip/spi_device/rtl/*
+hw/ip/spi_host/rtl/*
+hw/ip/sram_ctrl/rtl/*
+hw/ip/sysrst_ctrl/rtl/*
+hw/ip/tlul/rtl/*
+hw/ip/uart/rtl/*
+hw/ip/usbdev/rtl/*
+
+hw/ip_templates/alert_handler/rtl/*
+hw/ip_templates/clkmgr/rtl/*
```

Comment:
Especially for clkmgr and rstmgr but technically for all templated IP blocks here, we could drop blocker and instead enforce it on the top-level specific RTL and DV files. Because we will need changes to the templates and the generators to accommodate some of the changes required for Darjeeling (e.g., optionally generate SPI host / USB clocks etc.). But we don't really care about the template files changing as long as the generated RTL and DV files remain stable.

Reply:
Good point I'll drop the block on the templates